react/features/base/react/components/web/Switch.js

```diff
@@ -0,0 +1,62 @@
+/* @flow */
+
+import { ToggleStateless } from '@atlaskit/toggle';
+import React, { Component } from 'react';
+
+type Props = {
+
+    /**
+     * Handler called when the user presses the switch.
+     */
+    onValueChange: Function,
+
+    /**
+     * Indicates whether the switch is disabled or not.
+     */
+    disabled: boolean,
+
+    /**
+     * The current value.
+     */
+    value: boolean,
+
+    /**
+     * CSS class name.
+     */
+    className: string
+};
+
+/**
+ * Renders a boolean input.
+ */
+export default class Switch extends Component<Props> {
```

Comment:
Just a note that it usually makes it easier to review when changes that introduce new components are put into separate commits.